Esri/arcgis-runtime-samples-ios
# New: `Set up location-driven geotriggers`

## Description

Epic issue at `common-samples/issues/2661`

The map used in this sample: https://www.arcgis.com/home/item.html?id=6ab0e91dc39e478cae4f408e1a36a308

https://user-images.githubusercontent.com/9660181/129987868-016eeb95-b9e8-4c6a-bae2-fb0bd781cffa.mp4

This PR implements the Geotriggers sample for Update 12. Features and notes

- A mock walking tour around Santa Barbara Botanic Garden is used to show how fence geotriggers work in a real world scenario
- When a person walk into a garden section, the left button is enabled and they can check the detailed info of the section
- When a person walks nearby a point of interest in the garden, the right button is enabled, and they can see what's around
- If the person is inside multiple geofences, they can swipe back and forth in the popup VC to see multiple related info
- When a popup VC is presented, the location simulation will pause; when it is dismissed, the tour will resume

## How to test

Run the sample, read the README, and check if any UI would fall into a unexpected state.
